### PR TITLE
chore: add favicon to WDS Storybook

### PR DESCRIPTION
**Pull request by ichik**
tl;dr there was no favicon, added one. Slightly different fills for light and dark mode.

### CODE REVIEW

File: app/client/packages/storybook/favicon.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 16">
```
Comment (KelvinOm) [inline app/client/packages/storybook/favicon.svg:1]: Adding an icon to the root folder doesn't work, does it? We may do something like [this](https://github.com/storybookjs/storybook/issues/6155#issuecomment-474181062).

Comment (ichik) [reply app/client/packages/storybook/favicon.svg:1]: Hm, works locally for me. And markup to request was already there.

Screenshot 2023-08-22 at 14 30 20

Comment (KelvinOm) [reply app/client/packages/storybook/favicon.svg:1]: Let me check it. I also don't see favicon in the build [here](https://64c89da059b7d62ad3a60544-fhnunblgla.chromatic.com/?path=/docs/design-system-headless-button--docs).

Comment (ichik) [reply app/client/packages/storybook/favicon.svg:1]: Yeah, I guess it's the local FS that did the work.

Comment (KelvinOm) [reply app/client/packages/storybook/favicon.svg]: Pushed changes

Comment (KelvinOm) [reply app/client/packages/storybook/favicon.svg]: @ichik It works. Let's merge it? 

Снимок экрана 2023-08-23 в 10 29 22

Comment (ichik) [reply app/client/packages/storybook/favicon.svg]: Added some fixes (restored the actual custom icon that I created yesterday, removed duplicate markup, fixed incorrect mime type and sizes). Now show be good to merge.

**Review by KelvinOm — APPROVED** [review]

**Pull request merged**